_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,232 @@
+
+Navigate to your domain and check out your new site!
+
+![Example website deployed at example.matiaspan.dev](/images/2019-serverless-static-sites/hugo-example-site.png)
+
+## CI/CD with OpenFaaS Cloud
+[OpenFaaS Cloud](https://github.com/openfaas/openfaas-cloud) can offer you an automated experience, with a single push you can have your site deployed in no time!  
+If this sounds interesting to you check out our "single click" [bootstrap tool](https://github.com/openfaas-incubator/ofc-bootstrap) to install OpenFaaS Cloud in less than 100 seconds or apply for access to the [Community Cluster](https://github.com/openfaas/community-cluster).
+
+In OpenFaaS Cloud by default we limit the templates that the users can use. This means we need to add the hugo template before being able to deploy a function. To do this, edit the `git-tar` deployment in the `openfaas-fn` namespace and add the hugo template url `https://github.com/matipan/openfaas-hugo-template` to the `custom_templates` environment variable.
+
+Create a repository on GitHub(or your self hosted GitLab) called `example-site`. Add this repository to your GitHub app or add the `openfaas-cloud` tag if you are using GitLab.  
+Since we don't do recursive clones in OpenFaaS Cloud you will need to clone your site's theme instead of adding it as a submodule. Once you've done that commit and push your changes.  
+After a while, if you head over to your personal dashboard(find it at `system.<your domain>/dashboard/<username>`) you will see your new function deployed.  
+Edit the `FunctionIngress` so that it points to our brand new function `kubectl edit -n openfaas functioningresses.openfaas.com example-site`:
+```yaml
+apiVersion: openfaas.com/v1alpha2
+kind: FunctionIngress
+metadata:
+  name: example-site-tls
+  namespace: openfaas
+spec:
+  domain: "example.matiaspan.dev"
+  function: "<username>-example-site"
+  ingressType: "nginx"
+  tls:
+    enabled: true
+    issuerRef:
+      name: "letsencrypt-prod"
+      kind: "Issuer"
+```
+* You need to update the `function` to match with the new one. OpenFaaS Cloud deploys function using the following convention: `<username>-<function name>`. If you are unsure about the name you can find it with `faas-cli list`.
+
+We won't be using our previous function anymore so remove it:
+```sh
+faas-cli remove example-site
+```
+
+Try creating a new blog post with: `hugo new posts/my-first-post.md`. Remember to run that command inside the folder of your hugo site, not the root folder of the project.  
+Commit and push your changes again, after OpenFaaS Cloud does its thing you will be able to see your new changes deployed. Awesome!
+
+## CI/CD with GitLab
+If you can't or don't want to install OpenFaaS Cloud but you still want to have a [Netlify](https://netlify.com)-like experience then you can host your repository on GitLab and use their free CI/CD offerings. Save the following to a YAML file in the root folder of the repo `.gitlab-ci.yml`:
```

Comment:
This is the first time you've mentioned Netlify, there is no context to what Netlify is or why it would be wanted.

Perhaps in the intro to the blog post or this section on CI/CD, you could outline what a git-based experience looks and feels like? Reference Heroku also.

Reply:
Will do, thanks